### ISSUE TITLE
@vercel/og support needed ? 

### ISSUE THREAD

**Issue by nshelia** [magne4000/vite-plugin-vercel#23]
Getting following error when trying to use [`@vercel/og`](https://vercel.com/docs/functions/edge-functions/og-image-generation) with edge function defined at `endpoints/og.jsx`.

<img width="1492" alt="Screenshot 2023-09-24 at 5 42 55 PM" src="https://github.com/magne4000/vite-plugin-vercel/assets/27732608/59473dcc-0a59-4a0f-bcfb-baa0d0abd46a">

Vite version 4.2.1
Vite config

```

import { defineConfig, UserConfig } from 'vite';
import react from '@vitejs/plugin-react';
import ssr from 'vite-plugin-ssr/plugin';
import { fileURLToPath, URL } from 'url';
import vercel from 'vite-plugin-vercel';

// https://vitejs.dev/config/
export default defineConfig({
  plugins: [react(), ssr(), vercel()],

  resolve: {
    alias: [
      {
        find: '@',
        replacement: fileURLToPath(new URL('./src', import.meta.url)),
      },
    ],
  },
  vercel: {
    expiration: 25,
    additionalEndpoints: [
      {
        source: 'endpoints/og.jsx',
        destination: `edge`,
        edge: true,
        addRoute: true,
      },
    ],
  },
} as UserConfig);

```

**Comment by magne4000** [comment]
@nshelia should be fixed in `vite-plugin-vercel@0.3.3`

**Comment by nshelia** [comment]
Amazing! Thanks
